adafruit_platformdetect/chip.py

```diff
@@ -20,14 +20,27 @@
 """
 
 # imports
-from __future__ import annotations
+try:
+    from __future__ import annotations
+except ImportError:
+    pass
 import os
 import sys
-from typing import Optional, TYPE_CHECKING
+
+# from tokenize import Name
```

Comment:
MicroPython doesn't support the `tokenize` module, so I had to comment it out to get this working.  It appears to be an unused import and can probably be removed, but I wanted to double check that I didn't miss something.